db/upgrade.php

```diff
@@ -0,0 +1,36 @@
+function xmldb_qtype_myqtype_upgrade($oldversion) {
```

Comment:
Please add this comment to the top of the file, so it will match up with the upcoming development work.

```
/**
 * Credly's Acclaim Moodle Block Plugin
 * Credly: http://youracclaim.com
 * Moodle: http://moodle.org/
 *
 * @package    block_acclaim
 * @copyright  2020 Credly, Inc. <http://youracclaim.com>
 * @license    https://opensource.org/licenses/MIT
 */
```

Reply:
Done